### ISSUE TITLE
Injecting Context/Application in multi module app doesn't work

### ISSUE THREAD

**Issue by sudansh** [InsertKoinIO/koin-annotations#94]
**Describe the bug**
Injecting Context/Application in single module app works but doesn't work in multi module app

**To Reproduce**
1. Create multi module app

-  app
`startKoin{
           androidContext(applicationContext)
}`

This works in app module
`@Single
fun prefsApp(context: Context) = Preferences(context)`

-  feature1
This doesn't work in feature module

`@Single
fun prefsFeature(context: Context) = Preferences(context)`

**Expected behavior**
prefsFeature should be able to inject `context`

**Koin project used and used version (please complete the following information):**
 `koin-annotations-bom version 1.3.0`

**Comment by arnaudgiuliani** [comment]
why this doesn't work. Can you provide mode details?

**Comment by sudansh** [comment]
Created a sample project with bug
https://github.com/sudansh/koinBug/

Error while building
`e: [ksp] --> Missing Definition type 'android.content.Context' for 'app.sudansh.domain.PreferencesHelper'. Fix your configuration to define type 'Context'.
e: Error occurred in KSP, check log for detail`

**Comment by sudansh** [comment]
I saw this issue https://github.com/InsertKoinIO/koin-annotations/issues/49 and I believe this is not possible yet now to pass `context` with annotations.

**Comment by arnaudgiuliani** [comment]
it's possible with `androidContext(...)` line in your `startKoin` section. But verification seems to look for it

**Comment by sudansh** [comment]
Have already put `androidContext()` in `startKoin` but context isn't found in multi module app. It works if `PreferenceHelper` is in same module as `startKoin`

**Comment by arnaudgiuliani** [comment]
I don't see why your other module couldn't find your android Context if it's been injected since your startKoin section?! 🤔 
Can you gives more details about how your are launching your other modules?

**Comment by arnaudgiuliani** [comment]
yes, definitely need to update this project with the new compiler option. Also linked to this PR: https://github.com/InsertKoinIO/koin-annotations/pull/101

**Comment by Adnan9011** [comment]
same issue
when using context for create a database in the data module, context doesn't work
:(

**Comment by arnaudgiuliani** [comment]
>same issue
when using context for create a database in the data module, context doesn't work
:(

is it a KMP app? are you injecting on Android part?

**Comment by Adnan9011** [comment]
> > same issue
> > when using context for create a database in the data module, context doesn't work
> > :(
> 
> is it a KMP app? are you injecting on Android part?

Hello Arnaudgiuliani,

I have been using the Koin annotation in my Android project, but unfortunately, it doesn't work in multi-module setups. I then switched to using Koin without annotations.

I like the annotation feature, it just didn't work for me in this case (multi module). In my open-source project with a multi-module app, I have successfully used Koin (without annotations), which has worked well for me.
